recipes/scrapy-playwright/LICENSE

```diff
@@ -0,0 +1,26 @@
+Copyright 2020 Eugenio Lacuesta
```

Comment:
It looks like this file is already included in the source distribution, so it can be removed from this PR.

Reply:
Yes, thank you. I have removed the license file.